src/test/java/spoon/test/reflect/meta/MetaModelTest.java

```diff
@@ -0,0 +1,235 @@
+Unhandled method signature: CtType#setVisibility(spoon.reflect.declaration.ModifierKind)
+Unhandled method signature: CtType#getMethod(spoon.reflect.reference.CtTypeReference,java.lang.String,spoon.reflect.reference.CtTypeReference[])
+Unhandled method signature: CtType#getMethod(spoon.reflect.reference.CtTypeReference,java.lang.String,spoon.reflect.reference.CtTypeReference[])
+Unhandled method signature: CtType#getMethod(java.lang.String,spoon.reflect.reference.CtTypeReference[])
+Unhandled method signature: CtType#getMethod(java.lang.String,spoon.reflect.reference.CtTypeReference[])
+Unhandled method signature: CtType#getMethodsAnnotatedWith(spoon.reflect.reference.CtTypeReference[])
+Unhandled method signature: CtType#getMethodsAnnotatedWith(spoon.reflect.reference.CtTypeReference[])
+Unhandled method signature: CtCase#insertEnd(spoon.reflect.code.CtStatement)
+Unhandled method signature: CtCase#insertEnd(spoon.reflect.code.CtStatementList)
+Unhandled method signature: CtCase#insertBegin(spoon.reflect.code.CtStatementList)
+Unhandled method signature: CtMethod#setVisibility(spoon.reflect.declaration.ModifierKind)
+Unhandled method signature: CtJavaDocTag#setType(java.lang.String)
+Unhandled method signature: CtJavaDocTag#setType(java.lang.String)
+Unhandled method signature: CtField#setVisibility(spoon.reflect.declaration.ModifierKind)
+Unhandled method signature: CtAnnotation#setElementValues(java.util.Map)
+Unhandled method signature: CtAnnotation#setElementValues(java.util.Map)
+Unhandled method signature: CtAnnotation#addValue(java.lang.String,java.lang.Object)
+Unhandled method signature: CtAnnotation#addValue(java.lang.String,java.lang.Object)
+Unhandled method signature: CtAnnotation#addValue(java.lang.String,spoon.reflect.code.CtLiteral)
+Unhandled method signature: CtAnnotation#addValue(java.lang.String,spoon.reflect.code.CtLiteral)
+Unhandled method signature: CtAnnotation#addValue(java.lang.String,spoon.reflect.code.CtNewArray)
+Unhandled method signature: CtAnnotation#addValue(java.lang.String,spoon.reflect.code.CtNewArray)
+Unhandled method signature: CtAnnotation#addValue(java.lang.String,spoon.reflect.code.CtFieldAccess)
+Unhandled method signature: CtAnnotation#addValue(java.lang.String,spoon.reflect.code.CtFieldAccess)
+Unhandled method signature: CtAnnotation#addValue(java.lang.String,spoon.reflect.declaration.CtAnnotation)
+Unhandled method signature: CtAnnotation#addValue(java.lang.String,spoon.reflect.declaration.CtAnnotation)
+Unhandled method signature: CtEnum#getMethod(java.lang.String,spoon.reflect.reference.CtTypeReference[])
+Unhandled method signature: CtEnum#getMethod(spoon.reflect.reference.CtTypeReference,java.lang.String,spoon.reflect.reference.CtTypeReference[])
+Unhandled method signature: CtCatchVariable#setType(spoon.reflect.reference.CtTypeReference)
+Unhandled method signature: CtCatchVariable#getType()
+Unhandled method signature: CtCatchVariable#setVisibility(spoon.reflect.declaration.ModifierKind)

```

Comment:
Maybe this list should be part of an issue instead of putting it inside the code?